component/attr_cache/attr_cache_test.go

```diff
@@ -71,14 +71,14 @@ func newTestAttrCache(next internal.Component, configuration string) *AttrCache
 }
 
 func getDirPathAttr(path string) *internal.ObjAttr {
-	objAttr := getPathAttr(path, defaultSize, fs.FileMode(defaultMode), true)
+	objAttr := GetPathAttr(path, defaultSize, fs.FileMode(defaultMode), true)
 	flags := internal.NewDirBitMap()
 	flags.Set(internal.PropFlagMetadataRetrieved)
 	objAttr.Flags = flags
 	return objAttr
 }
 
-func getPathAttr(path string, size int64, mode os.FileMode, metadata bool) *internal.ObjAttr {
+func GetPathAttr(path string, size int64, mode os.FileMode, metadata bool) *internal.ObjAttr {
```

Comment:
This doesn't need to be exported (you can lower-case the function name).
The same goes for all the helpers in this file, since they don't get used outside this package.